src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -253,50 +253,75 @@ class C {
     [Fact]
     public async Task OpenStringLiteral()
     {
-        await VerifyItemIsAbsentAsync(AddUsingDirectives("using System;", AddInsideMethod("string s = \"$$")), @"String");
-        await VerifyItemIsAbsentAsync(AddUsingDirectives("using System;", AddInsideMethod("string s = \"$$")), @"System");
+        var code = AddUsingDirectives("using System;", AddInsideMethod("string s = \"$$"));
+        await VerifyExpectedItemsAsync(code, [
+            CompletionTestExpectedResult.Absent("String"),
```

Comment:
The record's name itself is overly descriptive too, so I'll rename it to `ItemExpectation`. We could also consider the option to create static helper methods in `AbstractCompletionProviderTests` to also save on an extra qualification, although it would open up to autocompletion mistakes.